demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -198,6 +198,7 @@ class BaseContent(BaseNode):
     git_sha: Optional[str]
     old_base_content_object: Optional["BaseContent"] = None
     related_content_dict: dict = Field({}, exclude=True)
+    structure_errors: Optional[list[dict]] = Field(None, exclude=True)
```

Comment:
let's use some model, instead of just a dict, to access the fields easily. Create a simple one in `base_strict_model.py`

Reply:
Done